crates/sncast/src/helpers/braavos.rs

```diff
@@ -45,6 +48,53 @@ where
         self.block_id = block_id;
         self
     }
+
+    async fn sign_deployment(
+        &self,
+        tx_hash: FieldElement,
+    ) -> Result<Vec<FieldElement>, S::SignError> {
+        let signature = self.signer.sign_hash(&tx_hash).await?;
+
+        // You can see params here:
+        // https://github.com/myBraavos/braavos-account-cairo/blob/6efdfd597bb051e99c79a512fccd14ee2523c898/src/presets/braavos_account.cairo#L104
+        // Order of the params is important, you can see way and order of deserialization here:
+        // https://github.com/myBraavos/braavos-account-cairo/blob/6efdfd597bb051e99c79a512fccd14ee2523c898/src/presets/braavos_account.cairo#L141
+        let aux_data: Vec<FieldElement> = vec![
+            // account_implementation
+            self.class_hash,
+            // signer_type
+            FieldElement::ZERO,
+            // secp256r1_signer.x.low
+            FieldElement::ZERO,
+            // secp256r1_signer.x.high
+            FieldElement::ZERO,
+            // secp256r1_signer.y.low
+            FieldElement::ZERO,
+            // secp256r1_signer.y.high
+            FieldElement::ZERO,
+            // multisig_threshold
+            FieldElement::ZERO,
+            // withdrawal_limit_low
+            FieldElement::ZERO,
+            // fee_rate
+            FieldElement::ZERO,
+            // stark_fee_rate
+            FieldElement::ZERO,
+            // chain_id
+            self.chain_id,
```

Comment:
Where does it come from? It's not listed as a parameter in the function you linked above

Reply:
You can see why it is needed [here](https://github.com/myBraavos/braavos-account-cairo/blob/6efdfd597bb051e99c79a512fccd14ee2523c898/src/presets/braavos_base_account.cairo#L79). I added this as comment. As said before there are no docs 😭

Reply:
You could add a link next to that comment explaining why it is needed with that link